tsan_suppressions.txt

```diff
@@ -11,6 +11,8 @@ race:ServiceTrackerTestFixture_DefaultCustomizerServiceTrackerCloseRace_Test::Te
 # Race condition on access of p.second.waitFlag within wait_for, TSAN false-positive
 mutex:TestSerialBundleInstall
 race:TestSerialBundleInstall
+mutex:sample::ServiceComponentCA06::Updated
+race:sample::ServiceComponentCA06::Updated
```

Comment:
are these false positives? what were the nature of the issues flagged by tsan?

Reply:
Yes they are, I included them in the set of false positives identified to be linked to the geck above these lines. I can add another specification, but they are the same as the two lines above -- also a race condition on bundleRegistry::Install.